cpanfile

```diff
@@ -0,0 +1,13 @@
+requires 'Locale::Country';
+requires 'Locale::Country::Multilingual';
+requires 'perl', '5.006';
+
+on configure => sub {
+    requires 'ExtUtils::MakeMaker';
+};
+
+on build => sub {
```

Comment:
```suggestion
on test => sub {
```

Reply:
```t/00-compile.t ......... 1/2 Locale::Country will be removed from the Perl core distribution in the next major release. Please install it from CPAN. It is being used at /home/travis/build/binary-com/perl-Locale-Country-Extra/.build/e8kC7i0vbb/blib/lib/Locale/Country/Extra.pm, line 7.
Locale::Codes will be removed from the Perl core distribution in the next major release. Please install it from CPAN. It is being used at /home/travis/perl5/perlbrew/perls/5.28.1/lib/5.28.1/Locale/Country.pm, line 22.```
So I guess we need to add Local::Country into cpanfile and install it by cpm if necessary.